Resources/Prototypes/Entities/Objects/Specific/chemistry.yml

```diff
@@ -187,7 +187,12 @@
   - type: Sprite
     sprite: Objects/Specific/Chemistry/pills.rsi
     state: pill
-  - type: Pill
+  - type: Tag
+    tags:
+      - Pill
+  - type: Food
+    transferAmount: 1000
```

Comment:
This is kinda suss, IMO probably better to make the field nullable and then have a null value transfer the whole thing?

Reply:
Yeah that seems better.